src/app/view.tsx

```diff
@@ -108,17 +139,24 @@ export function App() {
       <BodyCursor isDrag={isDrag()} />
 
       <Show when={isVisible()}>
-        <InspectorRoot style={{width: `${width()}px`}}>
+        <InspectorRoot style={styles()}>
           <DragHandler onMouseDown={onDown}>...</DragHandler>
           <TabsContainer>
             <TabsHeader>
-              <For each={Object.entries(Tabs)} fallback={<div>Loading...</div>}>
-                {([key, tab]) => (
-                  <Tab data-active={currentTab() === key} onClick={() => changeTab(key as Key)}>
-                    {tab.title}
-                  </Tab>
-                )}
-              </For>
+              <Header onMouseDown={onHeaderMouseDown}>
+                <Logo>☄️</Logo>
```

Comment:
added a logo for the case when the inspector was resized to the smallest size to keep the possibility to move

<img width="492" alt="image" src="https://user-images.githubusercontent.com/10822601/203035820-5980f30e-a50a-4383-aeb2-f1761f2faeb8.png">